### DISCUSSION TITLE
Optimal PC Build Specs for GPU inferencing with YOLOv8

### DISCUSSION THREAD

**Discussion by tristanfivaz** [ultralytics/ultralytics#15376] [Q&A]
Hi all!

I want to say congrats and thankyou to the Ultralytics team! Everything you guys do is incredibly commendable!!

I have been tasked with getting the parts to build a setup utilised for the development (training) and deployment of (inferencing and testing) yolo models to optimise my workflow. Currently I use a paid version of Google Colab to train these, however it would be optimal to do this locally and with much higher speeds. I also need to do some other general tasks like 3d modelling etc.

Does anyone have any recommendations on what Nvidea Graphics Card or CPU I should use? There's a lot of options and its rather overwhelming :p

Thanks in advance!

**Comment by pderrenger** [comment]
@tristanfivaz thank you for your kind words! For optimal performance with YOLOv8, consider an NVIDIA GPU with ample CUDA cores and VRAM, such as the RTX 3080 or higher. For the CPU, a high-core count processor like the AMD Ryzen 9 or Intel i9 series would be beneficial. Ensure you have sufficient RAM (32GB or more) and fast storage (NVMe SSD). This setup should provide a robust environment for both training and inferencing. For more detailed guidance, you can refer to our [hardware recommendations](https://docs.ultralytics.com/guides/nvidia-jetson/).
